app/pom.xml

```diff
@@ -603,17 +588,6 @@
             </executions>
           </plugin>
 
-          <plugin>
-            <groupId>com.mycila</groupId>
-            <artifactId>license-maven-plugin</artifactId>
-            <executions>
-              <execution>
-                <id>basepom.default</id>
-                <phase />
-              </execution>
-            </executions>
-          </plugin>
-
```

Comment:
This disables license header check, why run that if we're not running any other checks?

Reply:
Restoring....